### ISSUE TITLE
Backup of influxDB throws ERR_CHILD_PROCESS_STDIO_MAXBUFFER

### ISSUE THREAD

**Issue by macpo** [simatec/ioBroker.backitup#747]
**Describe the bug**  
The backup of ioBroker, Javascript, Jarvis, Grafana on my NAS worked a month ago. Now the backup throws the error ERR_CHILD_PROCESS_STDIO_MAXBUFFER while the backup of influxDB is running. If the backup of influxDB is deactivated the backup works fine.

Adapter influxDB 3.1.8
InfluxDB v1.6.7~rc0

**To Reproduce**  
Steps to reproduce the behaviour:
1. Activate influxDB in backitup
2. Start backup

```log
2022-09-14 08:32:11.178 - info: backitup.0 (3216) starting. Version 2.4.12 in /opt/iobroker/node_modules/iobroker.backitup, node: v16.16.0, js-controller: 4.0.23
2022-09-14 08:32:11.328 - info: backitup.0 (3216) [iobroker] backup was activated at 02:00 every 1 day(s)
2022-09-14 08:38:57.213 - error: backitup.0 (3216) [iobroker/influxDB] RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
2022-09-14 08:38:57.214 - error: backitup.0 (3216) [iobroker] RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stdout maxBuffer length exceeded
2022-09-14 08:39:09.676 - error: host.futBroker Caught by controller[0]: <--- Last few GCs --->
2022-09-14 08:39:09.677 - error: host.futBroker Caught by controller[0]: [3216:0x3c88f80] 420545 ms: Mark-sweep (reduce) 959.1 (972.0) -> 957.0 (970.5) MB, 11.3 / 0.0 ms (+ 26.6 ms in 32 steps since start of marking, biggest step 10.7 ms, walltime since start of marking 71 ms) (average mu = 0.858, current mu = 0.591) alloca[3216:0x3c88f80] 420608 ms: Mark-sweep (reduce) 959.9 (972.5) -> 957.9 (972.5) MB, 13.6 / 0.0 ms (+ 25.2 ms in 12 steps since start of marking, biggest step 14.6 ms, walltime since start of marking 50 ms) (average mu = 0.775, current mu = 0.386) alloca
2022-09-14 08:39:09.678 - error: host.futBroker Caught by controller[0]: <--- JS stacktrace --->
2022-09-14 08:39:09.678 - error: host.futBroker Caught by controller[1]: FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
2022-09-14 08:39:09.678 - warn: host.futBroker instance system.adapter.backitup.0 terminated due to SIGABRT
2022-09-14 08:39:09.678 - info: host.futBroker instance system.adapter.backitup.0 terminated with code NaN ()
```

Debug log running in the UI (ended in the middle of a line)
There are 2154 complete lines `/opt/iobroker/backups/influxDB_2022_09_14*` and one which is cutted in the middle.
```
Started iobroker ...
[DEBUG] [mount] - noserverino option: false
[DEBUG] [mount] - cifs-mount command: "sudo mount -t cifs -o username=myUser,password=****,rw,file_mode=0777,dir_mode=0777,vers=3.1.1 //myNas/Sicherungen/futBroker/ /opt/iobroker/backups"
[DEBUG] [mount] - mount successfully completed
[DEBUG] [mount] - done
[DEBUG] [iobroker] - host.futBroker 4143 states saved
[DEBUG] [iobroker] - host.futBroker 9677 objects saved
[DEBUG] [iobroker] - Backup created: /opt/iobroker/backups/iobroker_2022_09_14-16_00_28_backupiobroker.tar.gz
[DEBUG] [iobroker] - done
[DEBUG] [influxDB] - InfluxDB-Backup started ...
[DEBUG] [influxDB] - Start InfluxDB Backup ...
[DEBUG] [influxDB] - InfluxDB Backup tmp directory created 
[DEBUG] [influxDB] - Try deleting the InfluxDB tmp directory
[DEBUG] [influxDB] - InfluxDB tmp directory was successfully deleted
[DEBUG] [influxDB] - 2022/09/14 16:02:15 backing up metastore to /opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/meta.00
2022/09/14 16:02:15 backing up db=futStats
2022/09/14 16:02:15 backing up db=futStats rp=global shard=19 to /opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/futStats.global.00019.00 since 0001-01-01T00:00:00Z
2022/09/14 16:02:16 backing up db=futStats rp=global shard=20 to /opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/futStats.global.00020.00 since 0001-01-01T00:00:00Z
...
2022/09/14 16:06:12 backing up db=futStats rp=global shard=4615 to /opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/futStats.global.04615.00 since 0001-01-01T00:00:00Z
2022/09/14 16:06:12 backing up db=futStats rp=global shard=4616 to /opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/futStats.global.04616.00 since 0001-01-01T00:00:00Z
2022/09/14 16:06:12 backing up db=futStats rp=global shard=4617 to /opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/futStats.global.04617.00 since 0001-01-01T00:00:00Z
2022/09/14 16:06:12 backup complete:
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.meta
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s19.tar.gz
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s20.tar.gz
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s21.tar.gz
...
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s2196.tar.gz
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s2197.tar.gz
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s2198.tar.gz
2022/09/14 16:06:12 	/opt/iobroker/backups/influxDB_2022_09_14-16_02_15_backupiobroker/20220914T140215Z.s2199.tar.gz
```

**Expected behaviour**  
The backup of influxDB shall be performed.

**Screenshots & Logfiles**  
If applicable, add screenshots and logfiles to help explain your problem.

**Versions:**  
 - Adapter version: 2.4.12
 - JS-Controller version: 4.0.23 
 - Node version: v16.16.0 
 - Operating system: Raspberry Pi OS

**Additional context**

**Comment by simatec** [comment]
How big is your database?

**Comment by macpo** [comment]
About 700 MB

**Comment by macpo** [comment]
I reduced the data and the error appears with 540 MB also.

**Comment by simatec** [comment]
I think we have a problem here, which is also related to nodejs.
When the buffer becomes too large, there are unfortunately always problems.

Even with large iobroker backups this problem occurs.

My suggestion would be that you split your data to several databases.

This way you would also have a higher reliability.

**Comment by simatec** [comment]
Fix MaxBuffer in v2.5.4
